src/pages/resultsView/ResultsViewURLWrapper.ts

```diff
@@ -218,4 +260,24 @@ export default class ResultsViewURLWrapper extends URLWrapper<
     public setComparisonSubTabId(tabId: ResultsViewComparisonSubTab) {
         this.updateURL({ comparison_subtab: tabId });
     }
+
+    @computed public get selectedEnrichmentEventTypes() {
+        if (this.query.comparison_selectedEnrichmentEventTypes) {
+            return JSON.parse(
+                this.query.comparison_selectedEnrichmentEventTypes
+            ) as (
+                | MutationEnrichmentEventType
+                | CopyNumberEnrichmentEventType
+            )[];
+        } else {
+            return undefined;
+        }
+    }
+
+    @autobind
+    public updateSelectedEnrichmentEventTypes(t: EnrichmentEventType[]) {
+        this.updateURL({
+            comparison_selectedEnrichmentEventTypes: JSON.stringify(t),
+        });
+    }
 }
```

Comment:
This section is shared with GroupComparisonURLWrapper. Would it make sense to make a bse-class?

Reply:
there is a slight difference - the URL parameter is different. this is part of the reason I decided to use an interface rather than include this in the base class

Reply:
OK!